### DISCUSSION TITLE
在 windows  按照bat里面的编译 ，只有生成 PluginAdapter.dll  ilogtai.exe ，怎么生成插件的dll呢

### DISCUSSION THREAD

**Discussion by jcy1001** [alibaba/ilogtail#698] [Help] [answered]
在 windows  按照bat里面的编译 ，只有生成 PluginAdapter.dll  ilogtai.exe ，怎么生成插件的dll呢

**Comment by dragonYang200** [comment, marked as answer]
可以看下https://github.com/dragonYang200/ilogtail/blob/windows_compiler/scripts/README.md
